src/test/ExtractionClient.test.ts

```diff
@@ -141,4 +141,78 @@ describe("Extraction Client", () => {
       "pass",
     )).to.be.true;
   });
+
+  it("Get History", async () => {
+    const returns1 = {
+      extractions: [1, 2],
+      // eslint-disable-next-line @typescript-eslint/naming-convention
+      _links: {
+        next: {
+          href: "url",
+        },
+      },
+    };
+    const returns2 = {
+      extractions: [3, 4],
+      // eslint-disable-next-line @typescript-eslint/naming-convention
+      _links: {
+        next: undefined,
+      },
+    };
+    fetchStub.withArgs("https://api.bentley.com/insights/reporting/datasources/iModels/-/extraction/history", "pass").resolves(returns1)
```

Comment:
Please avoid using dashes in place of guids. It's hard to understand what they mean.

Reply:
You pass 2 dashes into `getExtractionHistory` method - which dash is added to the url?

Reply:
Removed dashes for this test. Perhaps could fix all other tests in another PR.